core/jobs/src/main/java/com/cognifide/aet/job/common/datafilters/jserrorsfilter/JsErrorsFilter.java

```diff
@@ -74,34 +72,21 @@ public boolean apply(@Nullable JsErrorLog input) {
   }
 
   private boolean shouldFilterOut(JsErrorLog jse) {
-    return matchSourceFile(sourceFile, jse.getSourceName())
-            && matchStrings(errorMessage, jse.getErrorMessage())
-            && matchNumbers(line, jse.getLineNumber());
+    boolean sourceNotSpecified = sourceFile == null;
+    boolean sourceEquals = jse.getSourceName().equals(sourceFile);
+    boolean sourceEndsWith = jse.getSourceName().endsWith("/" + sourceFile);
+
+    return (sourceNotSpecified || sourceEquals || sourceEndsWith)
```

Comment:
please allow adding the ability to ignore specific source file not all files with the same name.
eg:
index.html have linked following scripts:
http://my-page.com/data/jquery.js
http://thirdparty.com/amazon/jquery.js
http://thirdparty.com/amazon/version/2/jquery.js
http://thirdparty.com/amazon/version/2-min/jquery.js
and a single error occur in "http://thirdparty.com/amazon/version/2-min/jquery.js"

currently, a user is unable to ignore just "http://thirdparty.com/amazon/version/2-min/jquery.js"